.editorconfig

```diff
@@ -46,6 +46,9 @@ dotnet_style_collection_initializer = true:error
 dotnet_style_coalesce_expression = true:error
 dotnet_style_null_propagation = true:error
 dotnet_style_explicit_tuple_names = true:error
+dotnet_style_allow_multiple_blank_lines_experimental=false:error

```

Comment:
I still don't agree with marking them as errors, as it doesn't block compiles anyways.